workflows/python/sdk/order-processor/app.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import threading
+from time import sleep
+from dapr.ext.workflow import WorkflowRuntime, DaprWorkflowClient
+from workflow_activities import order_processing_workflow, notify_activity, process_payment_activity, verify_inventory_activity, update_inventory_activity, requst_approval_activity
+from dapr.clients import DaprClient
+from model import InventoryItem, OrderPayload
+from util import get_address
+
+store_name = "statestore-actors"
+
+default_item_name = "cars"
+
+class WorkflowConsoleApp:    
+    def main(self):
+        print("*** Welcome to the Dapr Workflow console app sample!")
+        print("*** Using this app, you can place orders that start workflows.")
+        print("*** Ensure that Dapr is running in a separate terminal window using the following command:")
+        print("dapr run --dapr-grpc-port 4001 --app-id order-processor")
+        # Wait for the sidecar to become available
+        sleep(5)
+
+        address = get_address()
+        workflowRuntime = WorkflowRuntime()
+        workflowRuntime.register_workflow(order_processing_workflow)
+        workflowRuntime.register_activity(notify_activity)
+        workflowRuntime.register_activity(requst_approval_activity)
+        workflowRuntime.register_activity(verify_inventory_activity)
+        workflowRuntime.register_activity(process_payment_activity)
+        workflowRuntime.register_activity(update_inventory_activity)
+        workflowRuntime.start()
+
+        daprClient = DaprClient(address=f'{address["host"]}:{address["port"]}')
+        baseInventory = {}
+        baseInventory["paperclip"] = InventoryItem("Paperclip", 5, 100)
+        baseInventory["cars"] = InventoryItem("Cars", 15000, 100)
+        baseInventory["computers"] = InventoryItem("Computers", 500, 100)
+
+        self.restock_inventory(daprClient, baseInventory)
+
+        client = DaprWorkflowClient(address["host"], address["port"])
```

Comment:
Now that the management portion of the SDK has been merged, we can use the regular DaprClient() rather than the DaprWorkflowClient.

Reply:
used